src/sensordatainterface/urls/edit_urls.py

```diff
@@ -78,4 +78,9 @@
     url(r'^actions/create-retrieval/(?:(?P<retrieval_id>\d+)/)?$', edit_views.edit_retrieval, name='create_retrieval'),
     url(r'^actions/create-retrieval/from_deployment/(?:(?P<deployment_id>\d+)/)?$', edit_views.edit_retrieval,
         name='create_retrieval_from_deployment'),
-]
+    url(r'^actions/deployment-detail/(?:(?P<action_id>\d+))/delete_action?$', edit_views.delete_action, name='delete_action'),

```

Comment:
since they're all accepting the same thing and have the same name, maybe all these can be something like

`url(r'^actions/delete/(?P<action_id>\d+)?$', edit_views.delete_action, name='delete_action'),`